frontend/Github/GithubIssue.svelte

```diff
@@ -214,18 +244,65 @@
                     </div>
                 {/if}
                 {#if aggregated}
-                    <div class="d-flex justify-content-end">
-                        <div class="ms-1">Runs:</div>
-                        {#each issue.runs as run, idx}
-                            <a class="ms-1" href="/test/{run.test_id}/runs?additionalRuns[]={run.run_id}">[{idx + 1}]</a>
-                        {/each}
+                    <div class="text-end my-2">
+                        <button class="btn btn-primary" on:click={() => (showRuns = true)}>View {issue.runs.length} runs</button>
                     </div>
                 {/if}
             </div>
         </div>
     </div>
 </div>
 
+{#if showRuns}
+    <div class="issue-run-list-modal">
+        <div class="d-flex align-items-center justify-content-center p-4">
+            <div class="rounded bg-white p-4 h-50">
+                <div class="mb-2 d-flex border-bottom pb-2">
+                    <h5>Runs for <span class="fw-bold">{issue.title}</span></h5>
+                    <div class="ms-auto">
+                        <button 
+                            class="btn btn-close"
+                            on:click={() => {
+                                showRuns = false;
+                            }}
+                        ></button>
+                    </div>
+                </div>
+                <div>
+                </div>
+                    {#await resolveRuns(issue.runs)}
```

Comment:
this is requested only when `showRuns` is true?

Reply:
Yes